internal/rbd/rbd_util.go

```diff
@@ -539,7 +540,7 @@ func (rv *rbdVolume) getTrashPath() string {
 func deleteImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) error {
 	image := pOpts.RbdImageName
 
-	util.DebugLog(ctx, "rbd: delete %s using mon %s, pool %s", image, pOpts.Monitors, pOpts.Pool)
+	log.ErrorLog(ctx, "rbd: delete %s using mon %s, pool %s", image, pOpts.Monitors, pOpts.Pool)
```

Comment:
this should be DebugLog? I see the same issue in other places also, copy-paste mistake?

Reply:
hmm, I'll check it again. Some search/replace seems to have gone wrong somehow...